tsconfig.json

```diff
@@ -13,8 +13,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true,
+    "incremental": true
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "scripts/generatePatientNames.ts"]
```

Comment:
Why do we need to exclude the script from tsconfig?

Reply:
Good question. I am excluding this because the scripts should not be included with the production build of the application.

When we run `next build`, it's going to take everything in `include` and use that as the typescript source files to generate the built application. The `scripts` directory should be explicitly excluded from this since it only used during development and not for the actual functionality in the application.

I was going to exclude the whole directory, but I realized that we're actually using a TypeScript interface exported from the parsePrimaryCodePath script in the source code, so excluding it will break the type defs. We should resolve this in a separate task

Reply:
Although I'm happy to tackle it here just lmk

Reply:
Thanks for the explanation :) I was curious as to why we would exclude the one script but not the other, so that explanation makes sense. You can address it here if you want but I am also fine with it being a separate task 👍